src/ucm/util/replace.c

```diff
@@ -126,30 +87,80 @@ UCM_OVERRIDE_FUNC(cudaHostUnregister,        cudaError_t)
 
 #endif
 
-#if HAVE___CURBRK
-extern void *__curbrk;
+#if UCM_BISTRO_HOOKS
```

Comment:
pls add comments to '#else' and '#endif' of UCM_BISTRO_HOOKS
e.g `#else  /* UCM_BISTRO_HOOKS */`

Reply:
added